source/scripting/Player.cpp

```diff
@@ -105,6 +106,14 @@ namespace GTA
 	{
 		return Native::Function::Call<Vehicle ^>(Native::Hash::GET_PLAYERS_LAST_VEHICLE);
 	}
+	int Player::MaxArmour::get()
+	{
+		return Native::Function::Call<int>(Native::Hash::GET_PLAYER_MAX_ARMOUR, Handle);
+	}
+	void Player::MaxArmour::set(int value)
+	{
+		Native::Function::Call(Native::Hash::SET_PLAYER_MAX_ARMOUR, Handle, value);
+	}
```

Comment:
American english would be good, I know it's hard for the british people :stuck_out_tongue_winking_eye: